source/chapters/configuration.rst

```diff
@@ -136,50 +136,136 @@ computer. **Therefore it is important to take care to choose the best Audio API
+  with the decks.
+
+* **Equalizer Plugin**: Here you can select the effect that is used as mixing
+  EQ in each deck. By default only built-in equalizers are displayed.  If you uncheck
+  "Only allow EQ knobs to control EQ specific effects" you can select any other
+  effect.
+
+
+  If you want different settings for each deck, you may uncheck "Use the same
+  EQ filter for all decks"
+
+* **Quick Effect**: Here you can select the effect that is controlled by the
+  dedicated filter knob in each deck. By default only built-in filter effects are
+  selected for all decks, but that can be changed as above.
+
+* **High/Low Shelf EQ**: This slider sets the crossover frequencies of the mixing 
+  EQ. It controls which frequency range is affected by the three Channel EQ
+  knobs Low, Mid and High. By default the low knob controls the bass and sub bass 
+  range up to 246 Hz. The mid knob controls the mid range up to 2,5 kHz. 
+  The remaining teble range is controlled by the high knob.   
+
+* **Master EQ**: This section allows you to setup an EQ that effects only the
+  master output. 
+
+
+Mixing Equalizers
+-----------------
+
+Mixxx offers three types of mixing equalizers with full kill feature. They are 
+actually isolators, adopted from analog cross over networks. Each EQ is 
+combination of a high shelf filter, a band pass filter, and a low shelf filter, 
+offering its own individual sound, so try them out.
+
+
+The Bessel EQs with Lipshitz and Vanderkooy Mix do not alter audio samples when
+all knobs are at center. Once you adjust the knobs it activates and requires more
+CPU time.
+
+The Linkwitz-Riley EQ always applies a minimum, natural sounding phase shift.
+The amount of CPU time does not change when you adjust the EQ knobs.
+
+The following table compares some technical parameters:
+
++----------------+--------+------------+-------------+-------------+-----------+
+| Type           | cut    | roll-off   | phase shift | bit perfect | CPU usage |
++================+========+============+=============+=============+===========+
+| Bessel4 LV-Mix | soft   | -24 db/Oct | linear      | yes         | low       |
++----------------+--------+------------+-------------+-------------+-----------+
+| Bessel8 LV-Mix | medium | -48 db/Oct | linear      | yes         | medium    |
++----------------+--------+------------+-------------+-------------+-----------+
+| Linkwitz-Riley | sharp  | -48 db/Oct | minimum     | no          | high      |
++----------------+--------+------------+-------------+-------------+-----------+
+
+cut: the frequency response (curve form) at the cross over frequency
+
+roll-off: The steepness of the EQ bands
+
+linear phase: No phase distortion, all frequencies are processed with the same group delay
+
+minimum phase: A natural phase distortion, the group delay changes by the frequency
+
+bit perfect: Whether the EQ changes the input at all when the EQ is at unity
```

Comment:
This is confusing since bit perfect "yes" means it's perfect. But this phrase makes it sounds like "yes" means the EQ changes the input when the EQ is at unity.